web/libs/editor/src/components/Comments/Comment/CommentFormButtons.scss

```diff
@@ -2,6 +2,8 @@
 .comment-form-buttons {
   &__buttons {
     display: flex;
+    height: 100%;
```

Comment:
On the icons branch this might conflict, just as a heads up. The comment form and buttons had style adjustments there to fix a few visual issues that were observed.